content/engineering/bruteforce-lock-and-unlock/index.md

```diff
@@ -0,0 +1,137 @@
+
+---
+
+## Basic Terminologies
+**Brute force attack:**
+- A method where every possible combination of characters or values is systematically tried to gain unauthorized access to a system, application, or data.
+
+**Brute force lock:**
+- Brute force lock is a type of account lock made to prevent a bruteforce attack.
+
+**Brute force lockout:**
+- Brute force lockout is a security mechanism that blocks access after a certain number of failed authentication attempts to prevent unauthorized access through repeated trial and error.
+
+**Multi-Factor Authentication (MFA):**
+-  MFA is a security method that requires multiple forms of identification to grant access, typically combining something the user knows, has, and is.
+
+In _LoginRadius_, the brute force lockout can be implemented in two ways, using:
+1. APIs
+2. Workflows
+
+> To implement brute force lockout in either of the above-mentioned ways, the user should own an admin account in the _LoginRadius Admin Console_.
+
+In this blog, we'll go through the API implementation of brute force lockout and user unlock
+
+---
+
+## API implementation
+### Creating a Basic Application
+1. In order to achieve brute force lockout using API, we need create a simple app with the login and registration features.
+2. This can be done by using the _Admin Console_.
+3. Navigate through ***Deployment->Identity Experience Framework***.
+4. Using theme, customization, preview, implement options, we can design the required application.
+5. If additional customization is required, the user can customize the predefined templates using the customize option, and then configure it as per the requirement.
+
+> The created app can be accessed using the link [https://`<app-name>`.devhub.lrinternal.com/auth.aspx](https://`<app-name>`.devhub.lrinternal.com/auth.aspx) in the _implement_ section of the _Identity Experience Framework_ or can be accessed from the preview section.
+
+### Brute force lockout implementation
+Brute force lockout can be achieved by following these steps: 
+1. Register with a new user, and login with the correct user email and password. It is observed that the user is redirected to the profile page after authentication.
+<br>
+[![loginpage_with_data.png](assets/loginpage_with_data.png)](assets/loginpage_with_data.png)
+<br>
+[![successful_login.png](assets/successful_login.png)](assets/successful_login.png)
+<br>
+
+2. Now, logout and try to login with incorrect credentials.
+3. If the password is incorrect successively till the lockout threshold, the account gets locked. 
+4. Therefore, brute force lockout is achieved.
+<br>
+[![incorrect_pwd.png](assets/incorrect_pwd.png)](assets/incorrect_pwd.png)
+<br>
+
+> In the _Admin Console_, we can set the _brute force lockout threshold, lockout type, and suspend effective period_. _LoginRadius_ supports _suspend, captcha, security question, and block type lockout_. 
+
+
+### Unlocking an account locked through Brute force lockout
```

Comment:
here you can elaborate more about types of brute force lockout loginradius support